algebird-core/src/main/scala/com/twitter/algebird/Group.scala

```diff
@@ -82,7 +87,16 @@ class ArrayGroup[T: ClassTag](implicit grp: Group[T])
   }.toArray
 }
 
-object Group extends GeneratedGroupImplicits with ProductGroups {
+class FromAlgebraGroup[T](m: AGroup[T]) extends FromAlgebraMonoid(m) with Group[T] {
```

Comment:
It would have to be an inner class of a trait to get the priority right. I think it is just clearer to manually handle it personally.